tests/test_gui/test_misc.py

```diff
@@ -52,6 +52,18 @@ def test_icon():
     assert result.status_code == 200
 
 
+def test_logo():
+    """Test that the home-page exits."""
+    app = create_app()
+
+    # sends HTTP GET request to the application
+    # on the specified path
+    result = app.get('/logo.png')
```

Comment:
@dishmop has this endpoint not been removed with the last commit?

Reply:
Yes - there is a different end point where the logo now sits, but the code doesn't know anything about it (it is just loaded directly by the html).